SaveMyEyes/sp.lproj/Localizable.strings

```diff
@@ -0,0 +1,23 @@
+/* 
+ Localizable.strings
+ SaveMyEyes
+ 
+ Creado por Max Omelchenko el 24/02/20.
+ Copyright © 2020 Max Omelchenko. Todos los derechos reservados.
+ */
+
+"Time to work" = “Hora de trabajar”;
```

Comment:
Please, use double quotes `"` instead of `“` here and across other translations as well.